src/wp-includes/media.php

```diff
@@ -1843,6 +1843,19 @@ function wp_filter_content_tags( $content, $context = null ) {
 				$filtered_image = wp_img_tag_add_loading_attr( $filtered_image, $context );
 			}
 
+			/**
+			 * Filters img tag that will be injected into the content.
```

Comment:
Nit-pick, but I'm not sure on the choice of word "injected" here. The filter is more about modifying an img tag that is already in the content. We may also want to clarify that this is not necessarily `the_content` - the `$context` is significant here.

Maybe something like this?
```suggestion
			 * Filters an img tag that within the content for a given context.
```